tools/check-commit-messages/check_commit_messages.py

```diff
@@ -73,6 +101,9 @@ def main():
             all_valid = False
 
     if not all_valid:
+        if not rebased:
```

Comment:
```suggestion
    if !(all_valid || not rebased):
```

Reply:
No, the idea is to show this message only if errors appear. 
If the branch is not rebased, it should not automatically fail the linter (to tell the truth, we have most of the PRs not fully rebased; it's a normal situation). We should just inform that it can be a problem if liner fails.

Reply:
ok